prisma/schema.prisma

```diff
@@ -145,3 +146,12 @@ model MagicLink {
   token      String   @unique
   expires    DateTime
 }
+
+model Invitation {
+  id           String   @id @default(cuid())
+  email        String
+  expires      DateTime
+  role         String   // future use
```

Comment:
Let's remove the `role` field for now since it won't be in use.  Once we get to that point we'll need to leverage the Privilege model as our project doesn't have 'roles' per se from the authorization sense.
```suggestion
```